diff --git a/gcovr/gcov.py b/gcovr/gcov.py
index f30215a..7fb0db2 100644
--- a/gcovr/gcov.py
+++ b/gcovr/gcov.py
@@ -156,7 +156,7 @@ def process_gcov_data(data_fname, covdata, source_fname, options):
     if filtered_fname is None:
         if options.verbose:
             sys.stdout.write("  Filtering coverage data for file %s\n" % fname)
-        return
+        # return
     #
     # Return if the filename matches the exclude pattern
     #
